bin/runtime-common/src/integrity.rs

```diff
@@ -316,6 +328,20 @@ pub fn check_message_lane_weights<C: Chain, T: frame_system::Config>(
 		this_chain_max_unrewarded_relayers,
 		this_chain_max_unconfirmed_messages,
 	);
+
+	// check that extra weights of delivery/confirmation transactions include the weight
+	// of `RefundBridgedParachainMessages` operations. This signed assumes the worst case
```

Comment:
I think this was supposed to be
```suggestion
	// of `RefundBridgedParachainMessages` operations. This signed extension assumes the worst case
```